packages/core/src/Footer/Footer.tsx

```diff
@@ -29,8 +29,8 @@ export interface HvFooterProps extends HvBaseProps {
  */
 export const HvFooter = (props: HvFooterProps) => {
   const {
-    name = "Hitachi Vantara",
-    copyright = `© Hitachi Vantara Corporation ${new Date().getFullYear()}. All Rights Reserved.`,
+    name = "Pentaho+",
```

Comment:
This will impact current DS5 users. Let's use the theme's defaultProps to change this. This way, each theme has their own "Brand name" automatically 😃

Reply:
oh yeah, I forgot about that. I'll update.